mkdocs.yml

```diff
@@ -43,7 +45,7 @@ extra:
       link: https://github.com/HGVSNomenclature/hgvs-nomenclature
 
 extra_css:
-  - stylesheets/extra.css
+  - stylesheets/extra.css?v=1
```

Comment:
This change doesn't seem to do anything, I think... :thinking:

Reply:
You're right of course. Will fix before merging.